projectroles/migrations/0026_delete_category_settings.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.18 on 2023-03-14 10:21
+
+from django.db import migrations
+from projectroles.app_settings import AppSettingAPI
+
+
+def clean_up_app_settings(apps, schema_editor):
+    app_settings_api = AppSettingAPI()
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+
+    # Find all app settings with project type 'CATEGORY'
+    app_settings = AppSetting.objects.filter(project__type='CATEGORY')
```

Comment:
I believe what we want to do here is:

- Query for all `AppSetting` objects where `project != None`
- Compare the project type against the app setting definition for that setting
    * IF `project.type not in setting_def.project_types`
        + OR `not setting_def.get('project_types') AND project.type == PROJECT_TYPE_CATEGORY`
        + OR definition is not found at all
    * THEN delete `AppSetting` object